### PR TITLE
[MSPAINT] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

File: base/applications/mspaint/lang/ro-RO.rc
Comment (binarymaster) [inline base/applications/mspaint/lang/ro-RO.rc:246]: Please wait, I'm going to improve wording of this line in English, it was added just today.

Comment (AndreiMiloiu) [reply base/applications/mspaint/lang/ro-RO.rc:246]: Hence, should I convert it to draft?

Comment (binarymaster) [reply base/applications/mspaint/lang/ro-RO.rc:246]: Maybe it's a good idea.

File: base/applications/mspaint/lang/ro-RO.rc
Comment (binarymaster) [inline base/applications/mspaint/lang/ro-RO.rc:246]: You can now update translation for this (and mark as ready for review).

Comment (AndreiMiloiu) [reply base/applications/mspaint/lang/ro-RO.rc:246]: Good, thank you!

File: base/applications/mspaint/lang/ro-RO.rc
Comment (reactos-contributor) [inline base/applications/mspaint/lang/ro-RO.rc:246]: Please use passive mood instead of reflexive one. Please see [Romanian translation notes](https://raw.githubusercontent.com/reactos/reactos/master/media/doc/Romanian%20translation%20notes.txt), point №20.

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

**Review by reactos-contributor — APPROVED** [review]

**Comment by HBelusca** [comment]
Note: this will need yet another update after PR #5357

**Comment by AndreiMiloiu** [comment]
So, should I reconvert it into a draft?

**Comment by HBelusca** [comment]
You may wait until that PR is merged (it will, very soon certainly)

**Comment by AndreiMiloiu** [comment]
Ok, thank you!

**Pull request merged**
